packages/react/src/components/LiveKitRoom.tsx

```diff
@@ -120,7 +120,7 @@ export function useLiveKitRoom(props: LiveKitRoomProps) {
 
   React.useEffect(() => {
     setRoom(passedRoom ?? new Room(options));
-  }, [options, passedRoom]);
+  }, [JSON.stringify(options), passedRoom]);
```

Comment:
actually not sure if this is ideal. we do have options that are probably not intended to be stringified... 
e.g. `expWebAudioMix: { audioContext: new AudioContext }`...

Reply:
Have you tried passing it the `new AudioContext`? Is it possible that `JSON.stringify` would throw an exception in this case?

Maybe it would be better if we wrap the options internally in a useState hook? Downside, you can only set the options once.

Reply:
audio context (as an example) gets stringified to `{}` which would be fine. Still slightly concerned about other options potentially having an impact here, but will have to look more closely through what we currently offer.

Reply:
checked again, stringifying should be fine!